### ISSUE TITLE
Free G729 request

### ISSUE THREAD

**Issue by starter48** [r3gis3r/CSipSimple#67]
Can you please add a free G729 codec into the codec pack?
[It is royalty free since 2017 year](http://www.sipro.com/G729.html).

**Comment by rustyx** [comment]
Please restore the G.729 codec, the patents have expired!!

The codec spec and a reference implementation are also [freely available](http://www.itu.int/rec/T-REC-G.729-199611-S!AnnA/en).

**Comment by russelltrafford** [comment]
Agreed, can this be added to csipsimple, or at least can instructions be given on how to manually install this into csipsimple? I mean there is no need for a g729 licence as of 2017, and its now 2018.

Would like the thoughts of the developers so we can at least see what you're thinking?

@r3gis3r ?
